mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(a) {
+  subTotal = a * .2;
```

Comment:
```suggestion
  let subTotal = a * .2;
```
When declaring your variables, you've ommitted the `let`/`const` keyword. This will still work but for reasons you'll learn about later this is bad practice ( you'll almost always declare variables with `let`/`const` )